content/events/2020-guadalajara/sponsor.md

```diff
@@ -3,64 +3,187 @@ Title = "Sponsor"
+          <tr>
+            <th scope="col">
+              <i>Packages</i>
+            </th>
+            <th scope="col">
+              <center>Community</center>
+            </th>
+            <th scope="col">
+              <center>Bronze</center>
+            </th>
+            <th scope="col">
+              <center>Silver</center>
+            </th>
+            <th scope="col">
+              <center>Gold</center>
+            </th>
+            <th scope="col">
+              <center>Platinum</center>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">Logo on website</th>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Social media shout-out</th>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+            <td>
+              <center>Yes</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Booth space</th>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center>Shared</center>
+            </td>
+            <td>
+              <center>Own</center>
+            </td>
+            <td>
+              <center>Own + Premium placement</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Full audience pitch</th>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center>Up to 1 minute</center>
+            </td>
+            <td>
+              <center>Up to 5 minutes</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Conference pass</th>
+            <td>
+              <center> </center>
+            </td>
+            <td>
+              <center>1</center>
+            </td>
+            <td>
+              <center>2</center>
+            </td>
+            <td>
+              <center>3</center>
+            </td>
+            <td>
+              <center>4</center>
+            </td>
+          </tr>
+          <tr>
+            <th scope="row">Price in USD*</th>
+            <td>
+              <center></center>
+            </td>
```

Comment:
Same as above - how about leaving the cell empty? e.g.
```
<td></td>
```

Reply:
oks used "empty cell" instead